.circleci/deploy.sh

```diff
@@ -44,35 +45,62 @@ export CLOUDSDK_CORE_DISABLE_PROMPTS=1
 curl https://sdk.cloud.google.com | bash
 ${HOME}/google-cloud-sdk/bin/gcloud --quiet components update
 
+echo "Configuring GitHub Container Repository configuration"
+echo ${GH_TOKEN_PUSH_TILLER} | docker login ghcr.io -u helm-bot --password-stdin
+
 echo "Configuring gcloud authentication"
 echo "${GCLOUD_SERVICE_KEY}" | base64 --decode > "${HOME}/gcloud-service-key.json"
 ${HOME}/google-cloud-sdk/bin/gcloud auth activate-service-account --key-file "${HOME}/gcloud-service-key.json"
 ${HOME}/google-cloud-sdk/bin/gcloud config set project "${PROJECT_NAME}"
 docker login -u _json_key -p "$(cat ${HOME}/gcloud-service-key.json)" https://gcr.io
 
-echo "Installing Azure CLI"
-apt update
-apt install -y apt-transport-https
-echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ stretch main" | tee /etc/apt/sources.list.d/azure-cli.list
-curl -L https://packages.microsoft.com/keys/microsoft.asc | apt-key add
-apt update
-apt install -y azure-cli
+echo "Configuring Docker Hub configuration"
+echo ${DOCKER_PASS} | docker login -u ${DOCKER_USER} --password-stdin
+
+echo "Configuring Quay configuration"
+echo ${QUAY_PASS} | docker login ghcr.io -u ${QUAY_USER} --password-stdin
```

Comment:
Here we're calling it "quay configuration", but we're logging into ghcr.io. This may be a mismatch, as on line 49 we've already logged into ghcr.io with another account.

Reply:
Fixed, along with some of the other issues @scottrigby found.